src/token-pool.js

```diff
@@ -39,7 +39,7 @@ class TokenPool {
   }
 
   async generateTokens() {
-    const accessToken = AccessToken.get();
+    const accessToken = await AccessToken.get();
```

Comment:
If this calls `AccessToken.refresh()` and the access token cookie is updated, `AccessToken.set` will call `generateTokens` (https://github.com/ghostery/ghostery-search-extension/blob/master/src/background.js#L11) so this will actually end up calling `generateTokens` twice.